packages/gatsby-admin/README.md

```diff
@@ -1,8 +1,28 @@
 # Gatsby Admin
 
-A visual interface to configure your Gatsby site.
+A visual interface to configure your Gatsby site. Currently alpha testing.
 
-## Architecture
+## Getting started
+
+To enable Admin for your local Gatsby site, run `gatsby develop` with the `GATSBY_EXPERIMENTAL_ENABLE_ADMIN` environment variable enabled:
```

Comment:
To lower the bar to entry we could/should make this more explicit. See the intro section of https://www.gatsbyjs.com/docs/file-system-page-creation/ which explains that you add this to your package.json and that Windows users should use `cross-env`

People already have their existing Gatsby sites and `scripts` in package.json, let's give them instructions to change that :)